python_modules/libraries/dagster-k8s/dagster_k8s/launcher.py

```diff
@@ -397,4 +397,6 @@ def check_run_worker_health(self, run: PipelineRun):
             )
         if job.status.failed:
             return CheckRunHealthResult(WorkerStatus.FAILED, "K8s job failed")
+        if job.status.succeeded:
+            return CheckRunHealthResult(WorkerStatus.SUCCESS)
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"startLine":401,"endLine":401}} -->
<i class='graphite__hidden'>[Re: lines 401 to 401]<br/><br/></i>

This has no effect on our current monitoring, but better to be accurate